### DISCUSSION TITLE
SocratiQ AI

### DISCUSSION THREAD

**Discussion by Mjrovai** [harvard-edge/cs249r_book#535] [General]
I would like to know how I can run SocratiQ AI locally. When rendering the book locally, it does not work. Also, if it is possible to use a smaller open model, a quantized LLama, for example). Thanks

**Comment by CPaes-BR** [comment]
Não posso te ajudar!
Sorry.

Em ter., 26 de nov. de 2024, 14:57, Marcelo Rovai ***@***.***>
escreveu:

**Comment by profvjreddi** [comment]
This is a good ask. @kai4avaya is still working on the documentation, and we will be sure to capture them there and possibly provide the ability to run things locally. A simple local endpoint API would suffice I would think.

**Comment by ghost** [comment]
@profvjreddi  Are there plans to port SocratiQ onto DeepSeek and would that potentially reduce the cost of scaling significantly? I am an entrepreneur and part-time high school teacher at an underserved high school in Chicago interested in democratizing access to premium education such as AP exam prep currently not affordable for many of our students. LinkedIn handle: tomslee99

**Reply by profvjreddi** [reply]
Hi @tomslee99 — thank you so much for reaching out and for your interest in SocratiQ. We haven’t seriously focused on cost of scaling yet, as we’re still in the beta stage and haven’t enabled it by default—most users aren’t actively using it just yet. That said, exploring alternative models like DeepSeek for inference efficiency is definitely on our radar, and it’s a great point to raise.

I’m especially grateful for the work you’re doing, i.e., bringing high-quality education to underserved communities is so important. 

If you’re exploring ways to introduce machine learning systems at the high school level, we’d love to hear more about your experience and any ideas you have. It’s a topic close to my heart, and I’d be excited to see how we might support that.

We have TinyML kits, etc., that we make available to interested folks to help teach, just in case you are interested.
